e2e/components/listing/details/criteria/criteria.po.js

```diff
@@ -0,0 +1,156 @@
+const elements = {
+  removedCriteria: '#removed-header',
+  attestToggle: '#success',
+  accept: '//span[text()="Accept"]',
+  testProcedureName: '#name',
+  version: '#version',
+  testToolsName: '#tt',
+  privacySecurityName: '#privacy-security-framework',
+};
+
+// Old elements added intentionally so it would be easy to clean up later
+const elementsOld = {
+  saveCertifiedProduct: 'button=Save Certification Criteria',
+  testProcedureName: '//*[@id="testProcedures "]',
+  testToolsName: '//*[@id="testTools "]',
+};
+
+class CriteriaComponent {
+  constructor() { }
+
+  get removedCriteriaHeader() {
+    return $(elements.removedCriteria);
+  }
+
+  editCriteria(id) {
+    $(`#criterion-id-${id}-edit`).scrollAndClick();
+  }
+
+  get attestToggle() {
+    return $(elements.attestToggle);
+  }
+
+  get accept() {
+    return $(elements.accept);
+  }
+
+  get saveCertifiedProductOld() {
+    return $(elementsOld.saveCertifiedProduct);
+  }
+
+  chipText(text) {
+    return $(`//span[text()="${text}"]`);
+  }
+
+  expandRemovedCriteria() {
+    $(elements.removedCriteria).$$('div')[1].scrollAndClick();
+  }
+
+  uiUpgradeFlag() {
+    if ($('chpl-certification-criteria').isExisting()) {
+      return false;
+    }
+    return true;
```

Comment:
This kind of construction:

```
if (something is true) {
  return false
}
return true
```

should almost always be replaced with just
```
return (something is false);
```

```suggestion
    return !$('chpl-certification-criteria').isExisting();
```